app/views/summary/index.html.haml

```diff
@@ -0,0 +1,43 @@
+%div.inset
+  %table.table.table-striped.table-bordered.table-responsive
+    %caption
+      %span.namespace= @summary.client_namespace
+      Requests: Fiscal Year
+      = @summary.fiscal_year
```

Comment:
Not sure I understand the benchmarking comment -- since `summary` does the current year, doesn't that provide a good benchmark for a year? 

I'd move to remove the `fiscal_year` use case until we are using it / get feedback from users that they want it